nightwatchJS/README.md

```diff
@@ -0,0 +1,159 @@
+
+<br />
+
+# NightwatchJS
+<!-- CONTENTS -->
+## Contents
+
+- [Intro](/nightwatchJS/Intro)
+- [Installation](/nightwatchJS/Installation)
+- [Write Tests](/nightwatchJS/WriteTest)
+- [How to Run](/nightwatchJS/HowtoRun)
+- [Reporting](/nightwatchJS/Reporting)
```

Comment:
This content menu is not redirecting properly, could you fix it?

Reply:
sure, let me fix